<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.1.189">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">


<title>BS3037: Virology - 8&nbsp; Plant and Protist Viruses</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1.6em;
  vertical-align: middle;
}
</style>


<script src="../site_libs/quarto-nav/quarto-nav.js"></script>
<script src="../site_libs/quarto-nav/headroom.min.js"></script>
<script src="../site_libs/clipboard/clipboard.min.js"></script>
<script src="../site_libs/quarto-search/autocomplete.umd.js"></script>
<script src="../site_libs/quarto-search/fuse.min.js"></script>
<script src="../site_libs/quarto-search/quarto-search.js"></script>
<meta name="quarto:offset" content="../">
<link href="../chapters/week2.html" rel="next">
<link href="../chapters/week11.html" rel="prev">
<script src="../site_libs/quarto-html/quarto.js"></script>
<script src="../site_libs/quarto-html/popper.min.js"></script>
<script src="../site_libs/quarto-html/tippy.umd.min.js"></script>
<script src="../site_libs/quarto-html/anchor.min.js"></script>
<link href="../site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="../site_libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" class="quarto-color-scheme" id="quarto-text-highlighting-styles">
<link href="../site_libs/quarto-html/quarto-syntax-highlighting-dark.css" rel="prefetch" class="quarto-color-scheme quarto-color-alternate" id="quarto-text-highlighting-styles">
<script src="../site_libs/bootstrap/bootstrap.min.js"></script>
<link href="../site_libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="../site_libs/bootstrap/bootstrap.min.css" rel="stylesheet" class="quarto-color-scheme" id="quarto-bootstrap" data-mode="light">
<link href="../site_libs/bootstrap/bootstrap-dark.min.css" rel="prefetch" class="quarto-color-scheme quarto-color-alternate" id="quarto-bootstrap" data-mode="dark">
<script id="quarto-search-options" type="application/json">{
  "location": "sidebar",
  "copy-button": false,
  "collapse-after": 3,
  "panel-placement": "start",
  "type": "textbox",
  "limit": 20,
  "language": {
    "search-no-results-text": "No results",
    "search-matching-documents-text": "matching documents",
    "search-copy-link-title": "Copy link to search",
    "search-hide-matches-text": "Hide additional matches",
    "search-more-match-text": "more match in this document",
    "search-more-matches-text": "more matches in this document",
    "search-clear-button-title": "Clear",
    "search-detached-cancel-button-title": "Cancel",
    "search-submit-button-title": "Submit"
  }
}</script>
<script async="" src="https://hypothes.is/embed.js"></script>


</head>

<body class="nav-sidebar floating">

<div id="quarto-search-results"></div>
  <header id="quarto-header" class="headroom fixed-top">
  <nav class="quarto-secondary-nav" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar" aria-controls="quarto-sidebar" aria-expanded="false" aria-label="Toggle sidebar navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
    <div class="container-fluid d-flex justify-content-between">
      <h1 class="quarto-secondary-nav-title"><span class="chapter-number">8</span>&nbsp; <span class="chapter-title">Plant and Protist Viruses</span></h1>
      <button type="button" class="quarto-btn-toggle btn" aria-label="Show secondary navigation">
        <i class="bi bi-chevron-right"></i>
      </button>
    </div>
  </nav>
</header>
<!-- content -->
<div id="quarto-content" class="quarto-container page-columns page-rows-contents page-layout-article">
<!-- sidebar -->
  <nav id="quarto-sidebar" class="sidebar collapse sidebar-navigation floating overflow-auto">
    <div class="pt-lg-2 mt-2 text-left sidebar-header">
    <div class="sidebar-title mb-0 py-0">
      <a href="../">BS3037: Virology</a> 
        <div class="sidebar-tools-main tools-wide">
    <a href="https://github.com/kevfo/BS3037_Notes/" title="Source Code" class="sidebar-tool px-1"><i class="bi bi-github"></i></a>
    <a href="" title="Share" id="sidebar-tool-dropdown-0" class="sidebar-tool dropdown-toggle px-1" data-bs-toggle="dropdown" aria-expanded="false"><i class="bi bi-share"></i></a>
    <ul class="dropdown-menu" aria-labelledby="sidebar-tool-dropdown-0">
        <li>
          <a class="dropdown-item sidebar-tools-main-item" href="https://twitter.com/intent/tweet?url=|url|">
            <i class="bi bi-bi-twitter pe-1"></i>
          Twitter
          </a>
        </li>
        <li>
          <a class="dropdown-item sidebar-tools-main-item" href="https://www.facebook.com/sharer/sharer.php?u=|url|">
            <i class="bi bi-bi-facebook pe-1"></i>
          Facebook
          </a>
        </li>
    </ul>
  <a href="" class="quarto-color-scheme-toggle sidebar-tool" onclick="window.quartoToggleColorScheme(); return false;" title="Toggle dark mode"><i class="bi"></i></a>
  <a href="" class="quarto-reader-toggle sidebar-tool" onclick="window.quartoToggleReader(); return false;" title="Toggle reader mode">
  <div class="quarto-reader-toggle-btn">
  <i class="bi"></i>
  </div>
</a>
</div>
    </div>
      </div>
      <div class="mt-2 flex-shrink-0 align-items-center">
        <div class="sidebar-search">
        <div id="quarto-search" class="" title="Search"></div>
        </div>
      </div>
    <div class="sidebar-menu-container"> 
    <ul class="list-unstyled mt-1">
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../index.html" class="sidebar-item-text sidebar-link">Preface</a>
  </div>
</li>
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a class="sidebar-item-text sidebar-link text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-1" aria-expanded="true"><strong>PART I: LECTURE NOTES</strong></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-1" aria-expanded="true">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-1" class="collapse list-unstyled sidebar-section depth1 show">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../chapters/week1.html" class="sidebar-item-text sidebar-link"><span class="chapter-number">1</span>&nbsp; <span class="chapter-title">A Brief Overview of Virology</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../chapters/week3.html" class="sidebar-item-text sidebar-link"><span class="chapter-number">2</span>&nbsp; <span class="chapter-title">Strategies for Virus Entry</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../chapters/week3actual.html" class="sidebar-item-text sidebar-link"><span class="chapter-number">3</span>&nbsp; <span class="chapter-title">Strategies for Virus Replication and Transcription</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../chapters/week4.html" class="sidebar-item-text sidebar-link"><span class="chapter-number">4</span>&nbsp; <span class="chapter-title">Virus Assembly and Exit</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../chapters/week5.html" class="sidebar-item-text sidebar-link"><span class="chapter-number">5</span>&nbsp; <span class="chapter-title">Antiviral Drugs</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../chapters/week6.html" class="sidebar-item-text sidebar-link"><span class="chapter-number">6</span>&nbsp; <span class="chapter-title">Antiviral Strategies and Virus Vaccines</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../chapters/week11.html" class="sidebar-item-text sidebar-link"><span class="chapter-number">7</span>&nbsp; <span class="chapter-title">Emerging and Re-Emerging Viruses</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../chapters/week12.html" class="sidebar-item-text sidebar-link active"><span class="chapter-number">8</span>&nbsp; <span class="chapter-title">Plant and Protist Viruses</span></a>
  </div>
</li>
      </ul>
  </li>
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a class="sidebar-item-text sidebar-link text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-2" aria-expanded="true"><strong>PART 2: TUTORIAL NOTES</strong></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-2" aria-expanded="true">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-2" class="collapse list-unstyled sidebar-section depth1 show">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../chapters/week2.html" class="sidebar-item-text sidebar-link"><span class="chapter-number">9</span>&nbsp; <span class="chapter-title">Seeing Viruses</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../chapters/tut2.html" class="sidebar-item-text sidebar-link"><span class="chapter-number">10</span>&nbsp; <span class="chapter-title">Growing and Counting Viruses</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../chapters/week3tut.html" class="sidebar-item-text sidebar-link"><span class="chapter-number">11</span>&nbsp; <span class="chapter-title">Engineering Viruses</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../chapters/week5tut.html" class="sidebar-item-text sidebar-link"><span class="chapter-number">12</span>&nbsp; <span class="chapter-title">Routes of Virus Transmission and Consequences</span></a>
  </div>
</li>
      </ul>
  </li>
    </ul>
    </div>
</nav>
<!-- margin-sidebar -->
    <div id="quarto-margin-sidebar" class="sidebar margin-sidebar">
        <nav id="TOC" role="doc-toc" class="toc-active">
    <h2 id="toc-title">Table of contents</h2>
   
  <ul>
  <li><a href="#plant-viruses" id="toc-plant-viruses" class="nav-link active" data-scroll-target="#plant-viruses"><span class="toc-section-number">8.1</span>  Plant Viruses</a>
  <ul class="collapse">
  <li><a href="#transmission" id="toc-transmission" class="nav-link" data-scroll-target="#transmission"><span class="toc-section-number">8.1.1</span>  Transmission</a></li>
  <li><a href="#plant-responses" id="toc-plant-responses" class="nav-link" data-scroll-target="#plant-responses"><span class="toc-section-number">8.1.2</span>  Plant Responses</a></li>
  <li><a href="#in-humans" id="toc-in-humans" class="nav-link" data-scroll-target="#in-humans"><span class="toc-section-number">8.1.3</span>  In Humans?</a></li>
  </ul></li>
  <li><a href="#nucleocytoplasmic-large-dna-viruses" id="toc-nucleocytoplasmic-large-dna-viruses" class="nav-link" data-scroll-target="#nucleocytoplasmic-large-dna-viruses"><span class="toc-section-number">8.2</span>  Nucleocytoplasmic Large DNA Viruses</a>
  <ul class="collapse">
  <li><a href="#mimivirus" id="toc-mimivirus" class="nav-link" data-scroll-target="#mimivirus"><span class="toc-section-number">8.2.1</span>  Mimivirus</a></li>
  </ul></li>
  <li><a href="#virophages" id="toc-virophages" class="nav-link" data-scroll-target="#virophages"><span class="toc-section-number">8.3</span>  Virophages</a>
  <ul class="collapse">
  <li><a href="#replication" id="toc-replication" class="nav-link" data-scroll-target="#replication"><span class="toc-section-number">8.3.1</span>  Replication</a></li>
  </ul></li>
  <li><a href="#infecting-humans" id="toc-infecting-humans" class="nav-link" data-scroll-target="#infecting-humans"><span class="toc-section-number">8.4</span>  Infecting Humans?</a></li>
  </ul>
<div class="toc-actions"><div><i class="bi bi-github"></i></div><div class="action-links"><p><a href="https://github.com/kevfo/BS3037_Notes/edit/main/chapters/week12.qmd" class="toc-action">Edit this page</a></p><p><a href="https://github.com/kevfo/BS3037_Notes/issues/new" class="toc-action">Report an issue</a></p></div></div></nav>
    </div>
<!-- main -->
<main class="content" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default">
<div class="quarto-title">
<h1 class="title d-none d-lg-block"><span class="chapter-number">8</span>&nbsp; <span class="chapter-title">Plant and Protist Viruses</span></h1>
</div>



<div class="quarto-title-meta">

    
    
  </div>
  

</header>

<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/week12/virosphere.png" class="img-fluid figure-img" width="578"></p>
<p></p><figcaption class="figure-caption">Genomic Sequences of Viruses</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p>The virosphere, which encompasses all viruses, represents a vast and diverse domain of microbial life. Within this realm, there are approximately 1,400 known species of human pathogens, which includes viruses, bacteria, fungi, protozoa, and helminths. However, it’s crucial to recognize that human pathogens constitute a mere fraction of the overall microbial diversity on Earth.</p>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/week12/plant_virus.png" class="img-fluid figure-img" width="348"></p>
<p></p><figcaption class="figure-caption">Examples of Plant Viruses</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p>The thing about plant viruses is that they don’t appear as distinct as animal viruses, likely because of size constraints.</p>
<p>Estimates for the total number of microbial species on our planet vary widely, ranging from as low as 120,000 to potentially tens of millions or even higher. As of 2011, despite significant advancements in scientific research, we have only managed to sequence a minuscule portion, approximately 1 × 10−22%, of the total DNA on Earth. This means that our exploration and understanding of microbial diversity, including viruses, remain at a nearly infinitesimal level.</p>
<p>The vast majority of microbial life, including a myriad of viral species, remains unexplored and uncharacterized. The limited sampling of microbial diversity underscores the immense complexity and richness of the virosphere, as well as the ongoing challenges in comprehensively studying and cataloging the diverse array of viruses present in different environments on our planet.</p>
<section id="plant-viruses" class="level2" data-number="8.1">
<h2 data-number="8.1" class="anchored" data-anchor-id="plant-viruses"><span class="header-section-number">8.1</span> Plant Viruses</h2>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/week12/examples.png" class="img-fluid figure-img" width="644"></p>
<p></p><figcaption class="figure-caption">Examples of Plant Viruses and Their Names</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p>In the realm of plant virology, there are some intriguing entities known as subviral agents, which are smaller than typical viruses and possess only a subset of the properties normally associated with viruses. Viroids represent one such category of subviral agents. These are composed of short strands of nucleic acid, specifically RNA. Unlike traditional viruses, viroids lack a protein coating and rely on the host cell machinery, particularly RNA polymerase, for the replication of their RNA.</p>
<p><strong>Viroids</strong> are predominantly found in higher plants and are unique in that they are not known to produce proteins. The absence of a protein coat and the reliance on the host cell machinery make viroids distinct entities with a simplified structure compared to conventional viruses. The role of viroids in plant pathology includes causing diseases, and their existence raises interesting questions about their evolutionary origins and whether they might be relics of a primordial RNA world.</p>
<p>Another intriguing class of subviral agents in plants is represented by <strong>satellites</strong>, specifically virusoids. <strong>Virusoids</strong> are single-stranded RNA molecules that are unable to replicate independently and require another virus, known as a helper virus, to facilitate their replication. Similar to viroids, virusoids are not known to produce proteins themselves. Their dependence on a helper virus distinguishes them as subviral agents that rely on the machinery of another virus for their replication.</p>
<section id="transmission" class="level3" data-number="8.1.1">
<h3 data-number="8.1.1" class="anchored" data-anchor-id="transmission"><span class="header-section-number">8.1.1</span> Transmission</h3>
<p>Plant viruses employ various strategies for their transmission, and two main modes are vertical and horizontal transmission. In <strong>vertical transmission</strong>, the virus is passed from one generation of plants to the next. This can occur externally through contaminated seeds, where the virus is present on the seed coat, or internally through a contaminated embryo. When seeds carry the virus, the resulting plants can propagate the virus as they grow, contributing to the persistence of the viral infection within a plant population.</p>
<p>On the other hand, <strong>horizontal transmission</strong> involves the movement of the virus from one plant to another, often facilitated by external factors. Vector transmission is a common mechanism where insects or nematodes act as carriers, acquiring the virus from an infected plant and transmitting it to a healthy plant during feeding. Contaminated tools, such as pruning shears or other equipment, can also contribute to horizontal transmission when plant sap carrying the virus is inadvertently transferred from one plant to another. Additionally, experimental or laboratory practices, where researchers handle infected plant material, can result in the transmission of viruses between plants.</p>
<section id="insect-vectors" class="level4" data-number="8.1.1.1">
<h4 data-number="8.1.1.1" class="anchored" data-anchor-id="insect-vectors"><span class="header-section-number">8.1.1.1</span> Insect Vectors</h4>
<p>Virus transmission in plants can occur through various mechanisms, and while direct plant-to-plant transmission via plant abrasion and contact with contaminated plant materials is not common, the majority of plant virus transmission involves insect vectors. Insects such as aphids, whiteflies, leafhoppers, mealy bugs, and thrips play a crucial role in spreading viruses among plants.</p>
<p>The process typically begins when an insect feeds on the leaves or stems of an infected plant. During this feeding, the insect acquires the virus from the contaminated plant. Subsequently, when the infected insect feeds on an uninfected plant, it transmits the virus to the healthy plant. This transmission during feeding is a key mechanism in the spread of plant viruses in agricultural and natural settings.</p>
<p>Once a plant is infected with the virus, the virus has the ability to spread internally within the plant. This internal spread allows the virus to colonize various tissues, leading to systemic infection. Importantly, insects may feed multiple times, increasing the likelihood of virus transmission to uninfected plants at high multiplicities.</p>
<section id="examples" class="level5" data-number="8.1.1.1.1">
<h5 data-number="8.1.1.1.1" class="anchored" data-anchor-id="examples"><span class="header-section-number">8.1.1.1.1</span> Examples</h5>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/week12/transmit.png" class="img-fluid figure-img" width="296"></p>
<p></p><figcaption class="figure-caption">Virus Transmission Routes</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p>In the context of insect-mediated transmission of plant viruses, there are two main types: noncirculative (nonpersistent) and circulative (persistent) transmission.</p>
<p>In <strong>noncirculative transmission</strong>, exemplified by potyvirus infection in aphids, the virus attaches to the aphid stylet during feeding on an infected plant. Subsequently, during the next feeding cycle on a healthy plant, the virus is injected into the plant, and the insect is only capable of transmitting the virus for a short period, typically a few hours.</p>
<p>On the other hand, <strong>circulative transmission</strong>, demonstrated by luteovirus infection in aphids, involves the virus passing through the insect without replication. The virus travels through the hindgut to the hemocoel and then enters the salivary gland. Unlike noncirculative transmission, the virus persists in the insect host and can be transmitted for an extended period, often several days.</p>
<p>In the realm of plant virus transmission, a third category exists known as <strong>circulative (persistent) and propagative transmission</strong>. Tospoviruses provide an example of this mode of transmission. Unlike noncirculative transmission where the virus does not replicate within the insect vector, and circulative transmission where the virus passes through the vector without replication, circulative and propagative transmission involves the multiplication of the virus within the insect.</p>
<p>In the case of tospoviruses, which are transmitted by the circulative mode, studies have demonstrated that these viruses are capable of multiplying in their insect vectors. This means that the virus undergoes a more conventional infection cycle within the insect host. The ability of tospoviruses to propagate within the insect contributes to their efficient transmission to plants during subsequent feeding. This intricate interaction between the virus and its insect vector adds complexity to the understanding of plant virus transmission mechanisms and has implications for the epidemiology and control of tospovirus-related diseases in crops.</p>
</section>
</section>
<section id="getting-into-the-plant" class="level4" data-number="8.1.1.2">
<h4 data-number="8.1.1.2" class="anchored" data-anchor-id="getting-into-the-plant"><span class="header-section-number">8.1.1.2</span> Getting into the Plant</h4>
<p>Once a plant virus is mechanically introduced into a plant, either through insect feeding or other means, it initiates infection in the inoculated cells, typically within a leaf. This initial infection is crucial for the virus to spread from cell to cell within the inoculated leaf and subsequently invade the entire plant, a process known as <strong>systemic infection</strong>.</p>
<p>Plant cells are surrounded by walls that are interconnected by transwall channels called <strong>plasmodesmata</strong>. These plasmodesmata play a vital role in the movement of plant viruses. They serve as conduits that allow the virus to travel directly from one plant cell to another without the need for an extracellular phase. This direct cell-to-cell movement facilitates the efficient spread of the virus within the plant, enabling it to establish a systemic infection and affect various tissues as it progresses through the plant’s growth. Understanding these processes is essential for deciphering the dynamics of plant virus infections and developing strategies for their control.</p>
<p>In the realm of plant virology, all plant viruses possess a distinctive class of proteins known as <strong>movement proteins (MPs)</strong>. These proteins play a crucial role in modifying or altering plasmodesmata, the channels connecting plant cells, thereby facilitating the direct spread of the virus genome from one cell to another. The MP is indispensable for this process.</p>
<p>When a plant virus experiences a mutation in the gene responsible for encoding its movement protein, the virus can still replicate within individual plant cell protoplasts and assemble virus particles. However, the mutated virus becomes defective in its ability to spread within the inoculated leaf and establish a systemic infection in the host plant. This underscores the essential role of the movement protein in the virus’s capacity for cell-to-cell movement, which is a critical aspect of the infection process.</p>
</section>
</section>
<section id="plant-responses" class="level3" data-number="8.1.2">
<h3 data-number="8.1.2" class="anchored" data-anchor-id="plant-responses"><span class="header-section-number">8.1.2</span> Plant Responses</h3>
<p>Plants possess sophisticated antivirus responses to counteract the threat of viral infections, employing a combination of protein-based and RNA-based mechanisms. One prominent defense mechanism is <strong>RNA silencing</strong>, where plant dicer-like proteins generate small RNA molecules. Transcriptional gene silencing involves the methylation and repression of targeted DNA sequences, hindering the expression of viral genes. Additionally, post-transcriptional gene silencing utilizes small RNAs to suppress the translation of viral RNAs, effectively limiting the production of viral proteins within the plant.</p>
<p>Another crucial aspect of the plant’s antivirus arsenal involves the <strong>salicylic acid (SA)-mediated response</strong>. SA, a key signaling molecule, plays a central role in orchestrating defense responses. For instance, the detection of pathogen-associated molecular patterns (PAMPs), such as double-stranded RNA (dsRNA), can lead to the accumulation of SA. This triggers a cascade of antiviral responses, including the activation of defense genes, as part of a mechanism known as PAMP-Triggered Immunity (PTI).</p>
<p>While plants have evolved these robust antivirus strategies, plant viruses, in turn, have developed mechanisms to counteract and circumvent these defenses. This ongoing arms race between the plant host and viruses underscores the dynamic nature of the host-pathogen interaction and highlights the intricate strategies employed by both parties to ensure their survival and propagation. The constant evolution of these defense and counter-defense mechanisms shapes the complex dynamics of plant-virus interactions in the ever-evolving battle for dominance within the host organism.</p>
</section>
<section id="in-humans" class="level3" data-number="8.1.3">
<h3 data-number="8.1.3" class="anchored" data-anchor-id="in-humans"><span class="header-section-number">8.1.3</span> In Humans?</h3>
<p>The interaction of plant viruses with humans is primarily indirect, occurring through the consumption of plant material that may harbor these viruses. Plant viruses have been detected in various food items, and when humans and animals consume plant-based products, there is potential for exposure to these viruses. Additionally, plant viruses can be found in the waste of animals and humans after the consumption of contaminated plant material.</p>
<p>One example highlighting the potential interaction is the case of Tobacco Mosaic Virus (TMV), which is present in tobacco plants. TMV has demonstrated stability in the environment, particularly in smoked tobacco products. Active smokers, who inhale and ingest tobacco smoke, may have detectable levels of TMV in their saliva and lungs. Furthermore, studies have shown the presence of anti-TMV immunoglobulin G (IgG) antibodies in the serum of smokers, indicating that the immune system may mount a response against the plant virus.</p>
<p>This evidence suggests that plant viruses can elicit immune responses in both animals and humans. While the impact of these interactions on human health is an area of ongoing research, the detection of plant viruses in the context of human consumption and exposure underscores the need for further exploration into the potential implications of these interactions on human health and immunity.</p>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/week12/spreading.png" class="img-fluid figure-img" width="420"></p>
<p></p><figcaption class="figure-caption">Virus Spreading</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p>The detection of plant viruses in humans and animals raises questions about the potential role of these organisms in the dissemination of plant viruses in the environment, particularly through human and animal waste. It is indeed very likely that humans and animals can play a role in the transmission and spread of plant viruses, as the viruses present in consumed plant material may be excreted in waste and subsequently introduced into the environment.</p>
<p>However, it’s important to note that there is currently no evidence to suggest that plant viruses can cause diseases in humans. While plant viruses have been detected in various tissues and bodily fluids, their ability to infect and replicate within human cells and tissues has not been demonstrated. The immune system of humans appears to effectively handle exposure to plant viruses without resulting in illness.</p>
<p>The potential role of humans and animals in the environmental dissemination of plant viruses highlights the need for ongoing research to better understand the dynamics of these interactions and their implications for agriculture and public health. As of now, the available evidence suggests that while humans and animals may contribute to the spread of plant viruses, these viruses do not pose a direct health risk to humans.</p>
</section>
</section>
<section id="nucleocytoplasmic-large-dna-viruses" class="level2" data-number="8.2">
<h2 data-number="8.2" class="anchored" data-anchor-id="nucleocytoplasmic-large-dna-viruses"><span class="header-section-number">8.2</span> Nucleocytoplasmic Large DNA Viruses</h2>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/week12/ncDNA.png" class="img-fluid figure-img" width="611"></p>
<p></p><figcaption class="figure-caption">Examples of NCLDV</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p>The category of nucleocytoplasmic large DNA viruses (NCLDV) encompasses a diverse group of viruses that share common genomic and structural characteristics, suggesting a potential common viral ancestor. Classified within Baltimore group 1 (dsDNA), these viruses are characterized by large genomes and the presence of numerous genes involved in DNA-related processes, such as DNA replication and repair. A distinctive feature of many viruses within this family is their ability to replicate in both the host’s nucleus and cytoplasm.</p>
<p>These viruses often infect small eukaryotic organisms known as protists. The NCLDV family includes a range of viruses that exhibit unique characteristics, and their ability to infect protists adds to the diversity of host organisms involved in viral interactions. The study of nucleocytoplasmic large DNA viruses provides insights into the evolution and complexity of viruses, as well as their diverse strategies for replication and interaction with host cells.</p>
<section id="mimivirus" class="level3" data-number="8.2.1">
<h3 data-number="8.2.1" class="anchored" data-anchor-id="mimivirus"><span class="header-section-number">8.2.1</span> Mimivirus</h3>
<p>Mimivirus belongs to the family Mimiviridae and constitutes a singular genus within this family. The genus is primarily represented by a single identified species known as Acanthamoeba polyphaga mimivirus (APMV), commonly referred to as “mimivirus.” The discovery of APMV occurred serendipitously within the amoeba Acanthamoeba polyphaga. Initially, due to its unusual characteristics, the virus was mistakenly identified as a gram-positive bacterium, leading to its name “Mimivirus,” reflecting its ability to mimic microbial features.</p>
<p>Since the inadvertent discovery of APMV, various strains of mimiviruses have been identified. These strains are categorized into three distinct sublineages, reflecting the genetic diversity within the Mimivirus genus. The exploration of mimiviruses has provided valuable insights into the complexity and diversity of large DNA viruses, highlighting their unique characteristics and evolutionary significance within the viral world.</p>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/week12/mimivirus_traits.png" class="img-fluid figure-img" width="280"></p>
<p></p><figcaption class="figure-caption">Traits of the Mimivirus</figcaption><p></p>
</figure>
</div>
</div>
</div>
<section id="traits" class="level4" data-number="8.2.1.1">
<h4 data-number="8.2.1.1" class="anchored" data-anchor-id="traits"><span class="header-section-number">8.2.1.1</span> Traits</h4>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/week12/shape.png" class="img-fluid figure-img" width="152"></p>
<p></p><figcaption class="figure-caption">Shape of the Mimivirus</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p>The capsid of Acanthamoeba polyphaga mimivirus (APMV) exhibits distinct structural features. It has an approximate size of about 500 nanometers and is adorned with fibrils measuring between 120 and 140 nanometers in length. These fibrils collectively form a dense layer on the capsid, and they are extensively glycosylated. This glycosylation is essential for facilitating the attachment of APMV to amoebae, a process mediated through interactions with glycans.</p>
<p>The capsid itself adopts an icosahedral structure, except at one specific vertex where a unique and noteworthy feature is present. At this particular vertex, the capsid is covered by a distinctive five-branch starfish-shaped structure known as the ‘<strong>stargate</strong>.’ Importantly, this stargate structure is devoid of the fibrils that characterize the rest of the capsid. These structural elements contribute to the overall architecture of APMV and play a crucial role in its interactions with host amoebae, shedding light on the intricacies of viral attachment and entry mechanisms.</p>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/week12/underneath.png" class="img-fluid figure-img" width="238"></p>
<p></p><figcaption class="figure-caption">Underneath the Mimivirus’ Capsid</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p>Beneath the external capsid of Acanthamoeba polyphaga mimivirus (APMV), there exists an intricate structure. Enclosed by an inner lipid membrane and fibers, this structure is a spherical lipid bilayer compartment, measuring approximately 340 nanometers in size. Within this compartment resides the viral genome and a variety of proteins. Notably, the nucleocapsid possesses a significant depression that faces the ‘stargate’ on the capsid, creating a cavity. This feature indicates that the nucleocapsid maintains a fixed position relative to the external capsid, showcasing the precision and organization of the viral architecture.</p>
<p>The APMV particles harbor a complex composition, consisting of around 114 proteins. Interestingly, this accounts for only 12% of the predicted number of genes within the virus. Among the identified proteins are those involved in crucial functions such as DNA repair, virus structure, and protein modifications.</p>
</section>
<section id="compared-to-other-viruses" class="level4" data-number="8.2.1.2">
<h4 data-number="8.2.1.2" class="anchored" data-anchor-id="compared-to-other-viruses"><span class="header-section-number">8.2.1.2</span> Compared to Other Viruses</h4>
<p>Mimivirus boasts a genome that is notably large and intricate, setting it apart from many other viruses. Its genome is characterized by double-stranded DNA (dsDNA) and has an impressive size of approximately 1,200 kilobases (kb). Within this extensive genetic material, the genome is predicted to harbor 911 protein-encoding genes. To put this into perspective, it’s worth noting that even the smallest known bacterium, Mycoplasma genitalium, possesses a genome of 580 kb, containing approximately 470 protein-encoding genes.</p>
<p>In fact, the Mimivirus genome surpasses the size of several bacterial genomes currently documented in databases. This emphasizes the remarkable complexity and diversity of Mimivirus, underscoring its capacity to carry a vast array of genetic information and encode a multitude of proteins. The expansive genome of Mimivirus contributes to its unique position among viruses, challenging traditional notions of viral genomic simplicity.</p>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/week12/gene.png" class="img-fluid figure-img" width="292"></p>
<p></p><figcaption class="figure-caption">Mimivirus Genomes</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p>The genome of Acanthamoeba polyphaga mimivirus (APMV) is noteworthy for encoding unique proteins, setting it apart from other viruses. Among the distinctive proteins encoded by APMV genes are translation factors and aminoacyl tRNA synthetases. These components play crucial roles in the translation process, emphasizing the virus’s ability to manipulate cellular machinery for its own replication.</p>
<p>Moreover, the APMV genome goes beyond the typical viral genetic repertoire by encoding four different transfer RNAs (tRNAs). tRNAs are essential molecules involved in protein synthesis, serving as adapters that bring amino acids to the growing polypeptide chain during translation. The presence of these tRNAs in the APMV genome underscores the virus’s capacity to independently contribute to the cellular processes necessary for protein synthesis, further highlighting the intricacies of its molecular biology.</p>
</section>
</section>
</section>
<section id="virophages" class="level2" data-number="8.3">
<h2 data-number="8.3" class="anchored" data-anchor-id="virophages"><span class="header-section-number">8.3</span> Virophages</h2>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/week12/virophages.png" class="img-fluid figure-img" width="471"></p>
<p></p><figcaption class="figure-caption">Information about Virophages</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p>The discovery of virophages has added a layer of complexity to our understanding of viral interactions. Interestingly, giant viruses, such as Acanthamoeba polyphaga mimivirus (APMV), have been found to be susceptible to infection by smaller viruses known as virophages. Virophages are characterized by their small size, typically ranging from 35 to 74 nanometers, and exhibit an icosahedral structure.</p>
<p>This phenomenon of virophages infecting giant viruses highlights the intricate dynamics within the microbial world. It introduces a viral component that influences the replication and behavior of giant viruses, potentially impacting the ecological relationships within microbial communities. The study of virophages expands our knowledge of the diverse interactions and dependencies that exist among different types of viruses, contributing to the ongoing exploration of viral ecology and evolution.</p>
<section id="replication" class="level3" data-number="8.3.1">
<h3 data-number="8.3.1" class="anchored" data-anchor-id="replication"><span class="header-section-number">8.3.1</span> Replication</h3>
<p>Virophages exhibit a unique dependence on other viruses, particularly giant viruses like mimiviruses, for their replication. Unlike many viruses that can replicate independently within their host organisms, virophages are unable to do so in amoeba hosts on their own. Instead, virophages are reliant on the presence of specific viruses, such as mimiviruses, for their replication.</p>
<p>The first identified virophage, named <strong>Sputnik</strong>, was isolated alongside mamavirus, a member of the mimivirus group. In co-infection scenarios, Sputnik has been observed to reduce the infectivity of mamavirus within the amoeba host. Additionally, the presence of Sputnik during co-infection has been linked to aberrant particle morphology in mamavirus, suggesting that virophages can have a modulating effect on the replication and behavior of their giant virus hosts. This intricate relationship highlights the interconnected nature of viral interactions and the role that virophages play in influencing the dynamics of viral infections within amoebas.</p>
</section>
</section>
<section id="infecting-humans" class="level2" data-number="8.4">
<h2 data-number="8.4" class="anchored" data-anchor-id="infecting-humans"><span class="header-section-number">8.4</span> Infecting Humans?</h2>
<p>While giant viruses of amoebae, such as mimiviruses and marseilleviruses, have been detected in studies involving humans, there is currently no evidence to suggest that these viruses directly cause disease in humans.</p>
<p>Mimiviruses have been identified through serological, molecular, and culture studies in patients with pneumonia. The detection of mimiviruses in these studies raises questions about their potential interactions with the human respiratory system, but as of now, there is no conclusive evidence linking them to human disease.</p>
<p>Similarly, marseilleviruses have been found in stool samples taken from humans. However, despite their presence, there is currently no confirmed evidence that marseilleviruses lead to disease in humans.</p>
<p>The relationship between giant viruses of amoebae and human health is an area of ongoing research, and scientists continue to investigate the potential impact of these viruses on human biology and disease.</p>


</section>

</main> <!-- /main -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const disableStylesheet = (stylesheets) => {
    for (let i=0; i < stylesheets.length; i++) {
      const stylesheet = stylesheets[i];
      stylesheet.rel = 'prefetch';
    }
  }
  const enableStylesheet = (stylesheets) => {
    for (let i=0; i < stylesheets.length; i++) {
      const stylesheet = stylesheets[i];
      stylesheet.rel = 'stylesheet';
    }
  }
  const manageTransitions = (selector, allowTransitions) => {
    const els = window.document.querySelectorAll(selector);
    for (let i=0; i < els.length; i++) {
      const el = els[i];
      if (allowTransitions) {
        el.classList.remove('notransition');
      } else {
        el.classList.add('notransition');
      }
    }
  }
  const toggleColorMode = (alternate) => {
    // Switch the stylesheets
    const alternateStylesheets = window.document.querySelectorAll('link.quarto-color-scheme.quarto-color-alternate');
    manageTransitions('#quarto-margin-sidebar .nav-link', false);
    if (alternate) {
      enableStylesheet(alternateStylesheets);
      for (const sheetNode of alternateStylesheets) {
        if (sheetNode.id === "quarto-bootstrap") {
          toggleBodyColorMode(sheetNode);
        }
      }
    } else {
      disableStylesheet(alternateStylesheets);
      toggleBodyColorPrimary();
    }
    manageTransitions('#quarto-margin-sidebar .nav-link', true);
    // Switch the toggles
    const toggles = window.document.querySelectorAll('.quarto-color-scheme-toggle');
    for (let i=0; i < toggles.length; i++) {
      const toggle = toggles[i];
      if (toggle) {
        if (alternate) {
          toggle.classList.add("alternate");     
        } else {
          toggle.classList.remove("alternate");
        }
      }
    }
    // Hack to workaround the fact that safari doesn't
    // properly recolor the scrollbar when toggling (#1455)
    if (navigator.userAgent.indexOf('Safari') > 0 && navigator.userAgent.indexOf('Chrome') == -1) {
      manageTransitions("body", false);
      window.scrollTo(0, 1);
      setTimeout(() => {
        window.scrollTo(0, 0);
        manageTransitions("body", true);
      }, 40);  
    }
  }
  const isFileUrl = () => { 
    return window.location.protocol === 'file:';
  }
  const hasAlternateSentinel = () => {  
    let styleSentinel = getColorSchemeSentinel();
    if (styleSentinel !== null) {
      return styleSentinel === "alternate";
    } else {
      return false;
    }
  }
  const setStyleSentinel = (alternate) => {
    const value = alternate ? "alternate" : "default";
    if (!isFileUrl()) {
      window.localStorage.setItem("quarto-color-scheme", value);
    } else {
      localAlternateSentinel = value;
    }
  }
  const getColorSchemeSentinel = () => {
    if (!isFileUrl()) {
      const storageValue = window.localStorage.getItem("quarto-color-scheme");
      return storageValue != null ? storageValue : localAlternateSentinel;
    } else {
      return localAlternateSentinel;
    }
  }
  let localAlternateSentinel = 'default';
  // Dark / light mode switch
  window.quartoToggleColorScheme = () => {
    // Read the current dark / light value 
    let toAlternate = !hasAlternateSentinel();
    toggleColorMode(toAlternate);
    setStyleSentinel(toAlternate);
  };
  // Ensure there is a toggle, if there isn't float one in the top right
  if (window.document.querySelector('.quarto-color-scheme-toggle') === null) {
    const a = window.document.createElement('a');
    a.classList.add('top-right');
    a.classList.add('quarto-color-scheme-toggle');
    a.href = "";
    a.onclick = function() { try { window.quartoToggleColorScheme(); } catch {} return false; };
    const i = window.document.createElement("i");
    i.classList.add('bi');
    a.appendChild(i);
    window.document.body.appendChild(a);
  }
  // Switch to dark mode if need be
  if (hasAlternateSentinel()) {
    toggleColorMode(true);
  } 
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const clipboard = new window.ClipboardJS('.code-copy-button', {
    target: function(trigger) {
      return trigger.previousElementSibling;
    }
  });
  clipboard.on('success', function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    setTimeout(function() {
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  });
  function tippyHover(el, contentFn) {
    const config = {
      allowHTML: true,
      content: contentFn,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start'
    };
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      return note.innerHTML;
    });
  }
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const cites = ref.parentNode.getAttribute('data-cites').split(' ');
    tippyHover(ref, function() {
      var popup = window.document.createElement('div');
      cites.forEach(function(cite) {
        var citeDiv = window.document.createElement('div');
        citeDiv.classList.add('hanging-indent');
        citeDiv.classList.add('csl-entry');
        var biblioDiv = window.document.getElementById('ref-' + cite);
        if (biblioDiv) {
          citeDiv.innerHTML = biblioDiv.innerHTML;
        }
        popup.appendChild(citeDiv);
      });
      return popup.innerHTML;
    });
  }
});
</script>
<nav class="page-navigation">
  <div class="nav-page nav-page-previous">
      <a href="../chapters/week11.html" class="pagination-link">
        <i class="bi bi-arrow-left-short"></i> <span class="nav-page-text"><span class="chapter-number">7</span>&nbsp; <span class="chapter-title">Emerging and Re-Emerging Viruses</span></span>
      </a>          
  </div>
  <div class="nav-page nav-page-next">
      <a href="../chapters/week2.html" class="pagination-link">
        <span class="nav-page-text"><span class="chapter-number">9</span>&nbsp; <span class="chapter-title">Seeing Viruses</span></span> <i class="bi bi-arrow-right-short"></i>
      </a>
  </div>
</nav>
</div> <!-- /content -->



</body></html>