<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.1.189">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">


<title>BS3037: Virology - 7&nbsp; Emerging and Re-Emerging Viruses</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1.6em;
  vertical-align: middle;
}
</style>


<script src="../site_libs/quarto-nav/quarto-nav.js"></script>
<script src="../site_libs/quarto-nav/headroom.min.js"></script>
<script src="../site_libs/clipboard/clipboard.min.js"></script>
<script src="../site_libs/quarto-search/autocomplete.umd.js"></script>
<script src="../site_libs/quarto-search/fuse.min.js"></script>
<script src="../site_libs/quarto-search/quarto-search.js"></script>
<meta name="quarto:offset" content="../">
<link href="../chapters/week2.html" rel="next">
<link href="../chapters/week6.html" rel="prev">
<script src="../site_libs/quarto-html/quarto.js"></script>
<script src="../site_libs/quarto-html/popper.min.js"></script>
<script src="../site_libs/quarto-html/tippy.umd.min.js"></script>
<script src="../site_libs/quarto-html/anchor.min.js"></script>
<link href="../site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="../site_libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" class="quarto-color-scheme" id="quarto-text-highlighting-styles">
<link href="../site_libs/quarto-html/quarto-syntax-highlighting-dark.css" rel="prefetch" class="quarto-color-scheme quarto-color-alternate" id="quarto-text-highlighting-styles">
<script src="../site_libs/bootstrap/bootstrap.min.js"></script>
<link href="../site_libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="../site_libs/bootstrap/bootstrap.min.css" rel="stylesheet" class="quarto-color-scheme" id="quarto-bootstrap" data-mode="light">
<link href="../site_libs/bootstrap/bootstrap-dark.min.css" rel="prefetch" class="quarto-color-scheme quarto-color-alternate" id="quarto-bootstrap" data-mode="dark">
<script id="quarto-search-options" type="application/json">{
  "location": "sidebar",
  "copy-button": false,
  "collapse-after": 3,
  "panel-placement": "start",
  "type": "textbox",
  "limit": 20,
  "language": {
    "search-no-results-text": "No results",
    "search-matching-documents-text": "matching documents",
    "search-copy-link-title": "Copy link to search",
    "search-hide-matches-text": "Hide additional matches",
    "search-more-match-text": "more match in this document",
    "search-more-matches-text": "more matches in this document",
    "search-clear-button-title": "Clear",
    "search-detached-cancel-button-title": "Cancel",
    "search-submit-button-title": "Submit"
  }
}</script>
<script async="" src="https://hypothes.is/embed.js"></script>


</head>

<body class="nav-sidebar floating">

<div id="quarto-search-results"></div>
  <header id="quarto-header" class="headroom fixed-top">
  <nav class="quarto-secondary-nav" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar" aria-controls="quarto-sidebar" aria-expanded="false" aria-label="Toggle sidebar navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
    <div class="container-fluid d-flex justify-content-between">
      <h1 class="quarto-secondary-nav-title"><span class="chapter-number">7</span>&nbsp; <span class="chapter-title">Emerging and Re-Emerging Viruses</span></h1>
      <button type="button" class="quarto-btn-toggle btn" aria-label="Show secondary navigation">
        <i class="bi bi-chevron-right"></i>
      </button>
    </div>
  </nav>
</header>
<!-- content -->
<div id="quarto-content" class="quarto-container page-columns page-rows-contents page-layout-article">
<!-- sidebar -->
  <nav id="quarto-sidebar" class="sidebar collapse sidebar-navigation floating overflow-auto">
    <div class="pt-lg-2 mt-2 text-left sidebar-header">
    <div class="sidebar-title mb-0 py-0">
      <a href="../">BS3037: Virology</a> 
        <div class="sidebar-tools-main tools-wide">
    <a href="https://github.com/kevfo/BS3037_Notes/" title="Source Code" class="sidebar-tool px-1"><i class="bi bi-github"></i></a>
    <a href="" title="Share" id="sidebar-tool-dropdown-0" class="sidebar-tool dropdown-toggle px-1" data-bs-toggle="dropdown" aria-expanded="false"><i class="bi bi-share"></i></a>
    <ul class="dropdown-menu" aria-labelledby="sidebar-tool-dropdown-0">
        <li>
          <a class="dropdown-item sidebar-tools-main-item" href="https://twitter.com/intent/tweet?url=|url|">
            <i class="bi bi-bi-twitter pe-1"></i>
          Twitter
          </a>
        </li>
        <li>
          <a class="dropdown-item sidebar-tools-main-item" href="https://www.facebook.com/sharer/sharer.php?u=|url|">
            <i class="bi bi-bi-facebook pe-1"></i>
          Facebook
          </a>
        </li>
    </ul>
  <a href="" class="quarto-color-scheme-toggle sidebar-tool" onclick="window.quartoToggleColorScheme(); return false;" title="Toggle dark mode"><i class="bi"></i></a>
  <a href="" class="quarto-reader-toggle sidebar-tool" onclick="window.quartoToggleReader(); return false;" title="Toggle reader mode">
  <div class="quarto-reader-toggle-btn">
  <i class="bi"></i>
  </div>
</a>
</div>
    </div>
      </div>
      <div class="mt-2 flex-shrink-0 align-items-center">
        <div class="sidebar-search">
        <div id="quarto-search" class="" title="Search"></div>
        </div>
      </div>
    <div class="sidebar-menu-container"> 
    <ul class="list-unstyled mt-1">
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../index.html" class="sidebar-item-text sidebar-link">Preface</a>
  </div>
</li>
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a class="sidebar-item-text sidebar-link text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-1" aria-expanded="true"><strong>PART I: LECTURE NOTES</strong></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-1" aria-expanded="true">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-1" class="collapse list-unstyled sidebar-section depth1 show">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../chapters/week1.html" class="sidebar-item-text sidebar-link"><span class="chapter-number">1</span>&nbsp; <span class="chapter-title">A Brief Overview of Virology</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../chapters/week3.html" class="sidebar-item-text sidebar-link"><span class="chapter-number">2</span>&nbsp; <span class="chapter-title">Strategies for Virus Entry</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../chapters/week3actual.html" class="sidebar-item-text sidebar-link"><span class="chapter-number">3</span>&nbsp; <span class="chapter-title">Strategies for Virus Replication and Transcription</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../chapters/week4.html" class="sidebar-item-text sidebar-link"><span class="chapter-number">4</span>&nbsp; <span class="chapter-title">Virus Assembly and Exit</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../chapters/week5.html" class="sidebar-item-text sidebar-link"><span class="chapter-number">5</span>&nbsp; <span class="chapter-title">Antiviral Drugs</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../chapters/week6.html" class="sidebar-item-text sidebar-link"><span class="chapter-number">6</span>&nbsp; <span class="chapter-title">Antiviral Strategies and Virus Vaccines</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../chapters/week11.html" class="sidebar-item-text sidebar-link active"><span class="chapter-number">7</span>&nbsp; <span class="chapter-title">Emerging and Re-Emerging Viruses</span></a>
  </div>
</li>
      </ul>
  </li>
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a class="sidebar-item-text sidebar-link text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-2" aria-expanded="true"><strong>PART 2: TUTORIAL NOTES</strong></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-2" aria-expanded="true">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-2" class="collapse list-unstyled sidebar-section depth1 show">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../chapters/week2.html" class="sidebar-item-text sidebar-link"><span class="chapter-number">8</span>&nbsp; <span class="chapter-title">Seeing Viruses</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../chapters/tut2.html" class="sidebar-item-text sidebar-link"><span class="chapter-number">9</span>&nbsp; <span class="chapter-title">Growing and Counting Viruses</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../chapters/week3tut.html" class="sidebar-item-text sidebar-link"><span class="chapter-number">10</span>&nbsp; <span class="chapter-title">Engineering Viruses</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../chapters/week5tut.html" class="sidebar-item-text sidebar-link"><span class="chapter-number">11</span>&nbsp; <span class="chapter-title">Routes of Virus Transmission and Consequences</span></a>
  </div>
</li>
      </ul>
  </li>
    </ul>
    </div>
</nav>
<!-- margin-sidebar -->
    <div id="quarto-margin-sidebar" class="sidebar margin-sidebar">
        <nav id="TOC" role="doc-toc" class="toc-active">
    <h2 id="toc-title">Table of contents</h2>
   
  <ul>
  <li><a href="#human-pathogens" id="toc-human-pathogens" class="nav-link active" data-scroll-target="#human-pathogens"><span class="toc-section-number">7.1</span>  Human Pathogens</a>
  <ul class="collapse">
  <li><a href="#influenza-viruses" id="toc-influenza-viruses" class="nav-link" data-scroll-target="#influenza-viruses"><span class="toc-section-number">7.1.1</span>  Influenza Viruses</a></li>
  </ul></li>
  <li><a href="#variation-between-viruses" id="toc-variation-between-viruses" class="nav-link" data-scroll-target="#variation-between-viruses"><span class="toc-section-number">7.2</span>  Variation Between Viruses</a>
  <ul class="collapse">
  <li><a href="#antigenic-drift" id="toc-antigenic-drift" class="nav-link" data-scroll-target="#antigenic-drift"><span class="toc-section-number">7.2.1</span>  Antigenic Drift</a></li>
  <li><a href="#reassortment" id="toc-reassortment" class="nav-link" data-scroll-target="#reassortment"><span class="toc-section-number">7.2.2</span>  Reassortment</a></li>
  <li><a href="#alternative-reading-frames" id="toc-alternative-reading-frames" class="nav-link" data-scroll-target="#alternative-reading-frames"><span class="toc-section-number">7.2.3</span>  Alternative Reading Frames</a></li>
  </ul></li>
  <li><a href="#the-influenza-virus" id="toc-the-influenza-virus" class="nav-link" data-scroll-target="#the-influenza-virus"><span class="toc-section-number">7.3</span>  The Influenza Virus</a>
  <ul class="collapse">
  <li><a href="#human-transmission" id="toc-human-transmission" class="nav-link" data-scroll-target="#human-transmission"><span class="toc-section-number">7.3.1</span>  Human Transmission</a></li>
  <li><a href="#bird-transmissions" id="toc-bird-transmissions" class="nav-link" data-scroll-target="#bird-transmissions"><span class="toc-section-number">7.3.2</span>  Bird Transmissions</a></li>
  <li><a href="#swine-transmission" id="toc-swine-transmission" class="nav-link" data-scroll-target="#swine-transmission"><span class="toc-section-number">7.3.3</span>  Swine Transmission</a></li>
  <li><a href="#first-direct-transmission" id="toc-first-direct-transmission" class="nav-link" data-scroll-target="#first-direct-transmission"><span class="toc-section-number">7.3.4</span>  First Direct Transmission</a></li>
  </ul></li>
  <li><a href="#host-adaptations" id="toc-host-adaptations" class="nav-link" data-scroll-target="#host-adaptations"><span class="toc-section-number">7.4</span>  Host Adaptations</a>
  <ul class="collapse">
  <li><a href="#influenza-polymerase-complex" id="toc-influenza-polymerase-complex" class="nav-link" data-scroll-target="#influenza-polymerase-complex"><span class="toc-section-number">7.4.1</span>  Influenza Polymerase Complex</a></li>
  <li><a href="#cap-snatching" id="toc-cap-snatching" class="nav-link" data-scroll-target="#cap-snatching"><span class="toc-section-number">7.4.2</span>  Cap Snatching</a></li>
  <li><a href="#host-protein-mutations" id="toc-host-protein-mutations" class="nav-link" data-scroll-target="#host-protein-mutations"><span class="toc-section-number">7.4.3</span>  Host Protein Mutations</a></li>
  <li><a href="#receptor-usage" id="toc-receptor-usage" class="nav-link" data-scroll-target="#receptor-usage"><span class="toc-section-number">7.4.4</span>  Receptor Usage</a></li>
  <li><a href="#activating-ha-protein" id="toc-activating-ha-protein" class="nav-link" data-scroll-target="#activating-ha-protein"><span class="toc-section-number">7.4.5</span>  Activating HA Protein</a></li>
  </ul></li>
  </ul>
<div class="toc-actions"><div><i class="bi bi-github"></i></div><div class="action-links"><p><a href="https://github.com/kevfo/BS3037_Notes/edit/main/chapters/week11.qmd" class="toc-action">Edit this page</a></p><p><a href="https://github.com/kevfo/BS3037_Notes/issues/new" class="toc-action">Report an issue</a></p></div></div></nav>
    </div>
<!-- main -->
<main class="content" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default">
<div class="quarto-title">
<h1 class="title d-none d-lg-block"><span class="chapter-number">7</span>&nbsp; <span class="chapter-title">Emerging and Re-Emerging Viruses</span></h1>
</div>



<div class="quarto-title-meta">

    
    
  </div>
  

</header>

<p><strong>Re-emerging viruses</strong> were once a big deal for people’s health, causing significant problems. Over time, though, these issues went down. However, now they’re making a comeback and causing health problems again.</p>
<p>An <strong>emerging virus</strong> is a newly discovered virus that has the potential to spread more and make people sick. It’s like a virus that’s just been found and has the ability to increase in how often it shows up and lead to diseases.</p>
<p>Several factors play a role in the emergence of viruses:</p>
<p>Firstly, viruses can adapt over time through processes like mutation, which means they change in a way that allows them to spread more easily.</p>
<p>Secondly, climate change is another factor. As the Earth’s climate shifts, it can create conditions that favor the growth and spread of new viruses.</p>
<p>Thirdly, economic development activities such as deforestation and changes in land use can contribute to the emergence of viruses. When we alter the environment, it can affect the balance between viruses and their hosts.</p>
<p>Fourthly, rapid travel, especially between continents, can speed up the spread of viruses. People moving quickly from one place to another can carry viruses with them, leading to new outbreaks in different regions.</p>
<p>Fifthly, the breakdown of public health care systems, particularly during times of war or crisis, can create conditions that allow viruses to thrive and spread.</p>
<p>Additionally, factors like poverty and social inequality can also contribute to the emergence of viruses. Limited access to healthcare and poor living conditions can make it easier for viruses to affect communities.</p>
<p>Furthermore, the intentional use of viruses for harmful purposes, known as bioterrorism, is another factor that can lead to the emergence of dangerous viruses.</p>
<p>Lastly, advancements in bioscience, such as the cost associated with developing effective vaccines, can influence the emergence of viruses. The availability and affordability of preventive measures play a crucial role in controlling the spread of viruses.</p>
<section id="human-pathogens" class="level2" data-number="7.1">
<h2 data-number="7.1" class="anchored" data-anchor-id="human-pathogens"><span class="header-section-number">7.1</span> Human Pathogens</h2>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/week11/human.png" class="img-fluid figure-img" width="386"></p>
<p></p><figcaption class="figure-caption">Reservoirs of Human Hosts</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p>Human pathogens use different strategies to spread:</p>
<p>Some viruses persist in humans by staying in a chronic or latent state with no other reservoir. For example, the Herpes simplex virus operates this way.</p>
<p>Others don’t stick around in humans but need to keep infecting new human hosts to survive. Diseases like measles and poliomyelitis follow this strategy.</p>
<p>Then, there are viruses that mainly hang out in nonhuman hosts and only accidentally infect humans. Rabies is an example of a virus that works this way. It’s important to note that some viruses, like influenza and SARS, can adapt to spread between humans. This adaptation is a significant source for the emergence of new viruses that infect humans.</p>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/week11/pig.png" class="img-fluid figure-img" width="264"></p>
<p></p><figcaption class="figure-caption">Links Between Reservoirs and Intermediate Hosts</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p>It may not always be possible to remove the intermediate hosts (e.g., bats are important to the ecosystem, culling pigs would severely hurt a country’s economy, etc).</p>
<section id="influenza-viruses" class="level3" data-number="7.1.1">
<h3 data-number="7.1.1" class="anchored" data-anchor-id="influenza-viruses"><span class="header-section-number">7.1.1</span> Influenza Viruses</h3>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/week11/flu_virus_chars.png" class="img-fluid figure-img" width="401"></p>
<p></p><figcaption class="figure-caption">Characteristics (underlined) of the Flu Virus</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p>Influenza viruses serve as a notable example of a virus that constantly poses emerging, reemerging, and current threats to public health. Understanding the principles involved and related issues in dealing with influenza is crucial. It’s important to note that while the discussion focuses on influenza, the same principles can be applied to other emerging viruses, such as the SARS-CoV-2 virus responsible for COVID-19.</p>
<p>Influenza is considered an emerging threat because it undergoes changes, or mutations, allowing new strains to emerge regularly. These changes can make existing vaccines less effective, requiring scientists to continually update them to match the evolving virus. The virus can also reemerge, causing outbreaks or pandemics, as seen in historical events like the Spanish flu of 1918. Additionally, influenza remains a current threat, with seasonal flu outbreaks affecting communities worldwide.</p>
<p>The principles involved in addressing influenza include monitoring and understanding the virus’s ability to change, developing effective vaccines, and implementing measures for prevention and control. Issues related to influenza management encompass the need for global collaboration in surveillance, the challenges of predicting which strains will dominate in a given season, and the importance of timely and widespread vaccination efforts.</p>
<p>While the focus here is on influenza, these principles and issues are applicable to other emerging viruses, such as SARS-CoV-2. Just as with influenza, understanding the genetic variations, developing effective vaccines, and establishing robust public health measures are critical components in addressing and mitigating the impact of emerging viruses on a global scale.</p>
<section id="categorizing-influenza-viruses" class="level4" data-number="7.1.1.1">
<h4 data-number="7.1.1.1" class="anchored" data-anchor-id="categorizing-influenza-viruses"><span class="header-section-number">7.1.1.1</span> Categorizing Influenza Viruses</h4>
<p>In the world of influenza viruses, classification is a key tool for scientists to understand their diverse nature. These viruses are divided into types A, B, and C, based on differences in the antigens found in their nucleocapsid (NP) and matrix (M) proteins. An intriguing addition to this classification is the relatively new D subtype, identified in cattle. Fortunately, as of now, this subtype is not known to infect humans. This system of categorization allows researchers to discern specific characteristics of each influenza virus, paving the way for targeted studies and strategic prevention measures.</p>
<p>Another facet of influenza classification involves the host specificity of each virus type. Influenza A, with its broad reach, can infect avian, human, and various mammalian species, including swine. Influenza B, in contrast, primarily targets humans, while Influenza C is mainly associated with human infections. This distinction in host preferences is crucial in understanding the potential impact and transmission dynamics of different influenza viruses within various populations.</p>
<p>Delving deeper into the molecular realm, each influenza virus type boasts a distinct set of proteins. While these proteins often share functional similarities, they are expressed from different genomic segments. In the case of Influenza A and B viruses, there are 8 genomic segments, whereas Influenza C (and D) virus possesses 7 segments. This genetic diversity contributes to the adaptability and evolution of influenza viruses, influencing their ability to infect different hosts and evade immune responses.</p>
<p>Furthermore, scientists employ subtyping based on the antigenicity of surface spike proteins to gain a finer understanding of each influenza virus. This additional layer of classification allows researchers to discern specific strains within each virus type, aiding in the development of vaccines and targeted intervention strategies. In summary, the intricate classification of influenza viruses provides a roadmap for scientists, offering valuable insights into their behavior, potential threats, and avenues for effective control and prevention.</p>
</section>
<section id="structure-of-influenza-viruses" class="level4" data-number="7.1.1.2">
<h4 data-number="7.1.1.2" class="anchored" data-anchor-id="structure-of-influenza-viruses"><span class="header-section-number">7.1.1.2</span> Structure of Influenza Viruses</h4>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/week11/structure.png" class="img-fluid figure-img" width="311"></p>
<p></p><figcaption class="figure-caption">Structure of a Typical Influenza Virus</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p>The structure of influenza viruses is fascinatingly intricate. Encased in a lipid envelope, which is enriched in raft lipids and derived from the host cell, these viral particles are relatively small, ranging from 80 to 120 nanometers in diameter. What makes them visually distinctive are the protein spikes that radiate from the virus. Among these spikes, there’s the rod-shaped <strong>haemagglutinin (HA)</strong> and the mushroom-shaped <strong>neuraminidase (NA)</strong>.</p>
<p>Beneath this outer layer, the genome of the influenza virus consists of negative-sense RNA segments. An interesting aspect is that the virus carries its own <strong>RNA-dependent RNA polymerase (RDRP)</strong>. This polymerase is crucial for the virus to replicate its RNA. The NP, PB1, PB2, and PA proteins come together to form the viral RNP (ribonucleoprotein), creating a distinct structure known as the <strong>RNP</strong>. This structure plays a key role in the virus’s ability to hijack host cells for its own replication.</p>
<p>Within the viral envelope, there are low levels of other integral virus membrane proteins, including M2, NB, and CM2. These proteins contribute to various functions in the virus life cycle. Additionally, the virus matrix (M) protein underlies the virus envelope, providing structural support to the overall architecture of the influenza virus.</p>
</section>
</section>
</section>
<section id="variation-between-viruses" class="level2" data-number="7.2">
<h2 data-number="7.2" class="anchored" data-anchor-id="variation-between-viruses"><span class="header-section-number">7.2</span> Variation Between Viruses</h2>
<section id="antigenic-drift" class="level3" data-number="7.2.1">
<h3 data-number="7.2.1" class="anchored" data-anchor-id="antigenic-drift"><span class="header-section-number">7.2.1</span> Antigenic Drift</h3>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/week11/antigenic_drift.png" class="img-fluid figure-img" width="396"></p>
<p></p><figcaption class="figure-caption">Sites of Antigenic Drift</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p>“<strong>Antigenic drift</strong>” is a term used to describe minor changes in the antigenic nature of influenza viruses. These changes result from accumulated amino acid changes, with a primary focus on two key proteins: hemagglutinin (HA) and neuraminidase (NA). This phenomenon is relevant to influenza A, B, and C viruses.</p>
<p>Historical examples of antigenic drift include the H1N1 virus responsible for the 1918 Spanish flu, the H2N2 virus associated with the 1957 Asian flu, and the H3N2 virus linked to the 1968 Hong Kong flu. In the case of the HA protein, antigenic drift occurs through amino acid changes at five specific antigenic sites, giving rise to different influenza virus subtypes. These changes have a significant impact as they prevent the binding of antibodies generated from previous infections, making it challenging for the immune system to recognize and defend against the modified virus.</p>
<p>In human influenza A viruses, antibody selection plays a role in driving antigenic drift. The immune system’s response to previous infections influences the evolution of the virus by selecting for variants that can escape existing antibodies. Antigenic sites are also identified in the NA protein, adding another layer to the complexity of influenza virus evolution.</p>
</section>
<section id="reassortment" class="level3" data-number="7.2.2">
<h3 data-number="7.2.2" class="anchored" data-anchor-id="reassortment"><span class="header-section-number">7.2.2</span> Reassortment</h3>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/week11/reassortment.png" class="img-fluid figure-img" width="124"></p>
<p></p><figcaption class="figure-caption">Reassortment</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p><strong>Reassortment</strong> is a phenomenon observed in influenza A, B, and C viruses, where these viruses have the ability to create reassorted or mixed viruses. This occurs when two different influenza viruses infect the same host cell, leading to the exchange of genetic material between the two viruses.</p>
<p>An illustrative example of reassortment is seen in the influenza A virus A/Kong Kong/68 H3N2. In this case, the reassorted virus contains genes for the hemagglutinin (HA) and polymerase basic 1 (PB1) from an H3 virus and the neuraminidase (NA) and five other genes from a circulating H2N2 virus. This mixing of genetic material can result in a new virus with a combination of genetic elements from different parental viruses.</p>
</section>
<section id="alternative-reading-frames" class="level3" data-number="7.2.3">
<h3 data-number="7.2.3" class="anchored" data-anchor-id="alternative-reading-frames"><span class="header-section-number">7.2.3</span> Alternative Reading Frames</h3>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/week11/fragments.png" class="img-fluid figure-img" width="442"></p>
<p></p><figcaption class="figure-caption">Fragments</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p>Alternative reading frames, exemplified by the PB1-F2 protein, introduce an intriguing layer of complexity to influenza viruses. The <strong>PB1-F2 protein</strong> is encoded by alternative reading frames on a specific genomic segment and gives rise to both the traditional PB1 protein and the unique PB1-F2 protein. It’s important to note that this alternative reading frame is not present in all influenza viruses.</p>
<p>This distinctive reading frame has been identified in significant influenza pandemic strains, including those from 1918, 1957, and 1968. The PB1-F2 protein is proposed to play a role in inducing apoptosis, a process of programmed cell death, particularly in macrophages. This programmed cell death in macrophages, which are immune cells that play a key role in defense against infections, can have implications for the pathogenicity of the virus.</p>
<p>The impact extends to alveolar macrophages, which are crucial in defending the lungs. If these macrophages are impaired due to the actions of PB1-F2, it can contribute to the severity of the infection. Additionally, the PB1-F2 protein has been implicated in facilitating bacterial secondary infections, adding another layer of complexity to the interplay between the influenza virus and the host immune response. Understanding the role of alternative reading frames like PB1-F2 provides insights into the various strategies employed by influenza viruses to modulate host responses and influence the course of infection.</p>
</section>
</section>
<section id="the-influenza-virus" class="level2" data-number="7.3">
<h2 data-number="7.3" class="anchored" data-anchor-id="the-influenza-virus"><span class="header-section-number">7.3</span> The Influenza Virus</h2>
<section id="human-transmission" class="level3" data-number="7.3.1">
<h3 data-number="7.3.1" class="anchored" data-anchor-id="human-transmission"><span class="header-section-number">7.3.1</span> Human Transmission</h3>
<p>Human transmission of influenza primarily occurs through human-to-human contact, often facilitated by the spread of respiratory droplets in the form of aerosols. This mode of transmission is a key factor in the occurrence of both epidemics and pandemics of influenza.</p>
<p>Epidemics are characterized by the widespread occurrence of influenza within a specific region or community. Seasonal influenza, a familiar term, refers to the regular, often annual, occurrence of epidemics. The size and impact of these epidemics are closely linked to the immunologic status of the population. The concept of herd immunity, where a significant portion of the population becomes immune to a specific virus either through vaccination or previous infection, plays a crucial role in mitigating the spread of influenza during epidemics. The higher the level of immunity in the population, the more challenging it is for the virus to find susceptible individuals, thus limiting the scale of the epidemic.</p>
<section id="pandemics" class="level5" data-number="7.3.1.0.1">
<h5 data-number="7.3.1.0.1" class="anchored" data-anchor-id="pandemics"><span class="header-section-number">7.3.1.0.1</span> Pandemics</h5>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/week11/pandemics.png" class="img-fluid figure-img" width="275"></p>
<p></p><figcaption class="figure-caption">WHO’s Definition of a Pandemic</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p><strong>Pandemics</strong> in the context of influenza occur when a new subtype of the virus, not previously encountered by the existing human population, emerges. The critical characteristic distinguishing pandemics from regular epidemics is the lack or limited presence of preexisting immunity in the population. Unlike seasonal epidemics, where some level of immunity may exist due to previous exposure or vaccination, pandemics arise from influenza strains that are entirely new to the human immune system.</p>
<p>The degree of severity associated with a pandemic depends on the specific characteristics of the emerging strain. Some pandemic strains may exhibit mild symptoms and lower fatality rates, while others can be more severe and pose significant public health challenges. The unpredictability of the severity is a distinctive feature of pandemics, making it crucial for public health authorities to closely monitor and respond to emerging strains to mitigate their impact on the global population.</p>
</section>
<section id="antiviral-drugs-against-influenza-virus" class="level4" data-number="7.3.1.1">
<h4 data-number="7.3.1.1" class="anchored" data-anchor-id="antiviral-drugs-against-influenza-virus"><span class="header-section-number">7.3.1.1</span> Antiviral Drugs Against Influenza Virus</h4>
<p>Antiviral drugs play a crucial role in managing influenza, and two classes of such drugs are Amantadine/Zamantadine and Zanamivir/Oseltamivir (Tamiflu™).</p>
<p>Amantadine and Zamantadine are among the oldest antiviral medications used against influenza. They primarily target influenza A viruses by inhibiting the M2 protein. However, one significant challenge with these drugs is the development of resistance, limiting their effectiveness in some cases.</p>
<p>On the other hand, Zanamivir and Oseltamivir (Tamiflu™) belong to a relatively newer class known as neuraminidase inhibitors. These drugs work by mimicking the natural substrate, sialic acid, of the neuraminidase (NA) protein on the surface of influenza viruses. By doing so, they prevent the release of new virus particles from infected cells, hindering the spread of the virus. Notably, these drugs are effective against both influenza A and B viruses.</p>
<p>While Zanamivir and Oseltamivir have proven to be valuable in influenza treatment, it’s important to acknowledge the presence of resistance, both in laboratory settings (in vitro) and in real-world scenarios (in vivo). The development of resistance highlights the ongoing challenge of managing influenza and the need for continued research and the development of new antiviral strategies.</p>
</section>
<section id="vaccines" class="level4" data-number="7.3.1.2">
<h4 data-number="7.3.1.2" class="anchored" data-anchor-id="vaccines"><span class="header-section-number">7.3.1.2</span> Vaccines</h4>
<p>Vaccines are crucial tools in preventing and controlling the spread of influenza, but they require time for preparation. Traditionally, influenza vaccines have been developed based on the major circulating strains in a specific region. This involves predicting the most likely strains to be prevalent in the upcoming flu season.</p>
<p>Regarding the use of RNA vaccines, there has been significant progress in recent years, and RNA-based influenza vaccines are being explored. These vaccines use genetic material, such as messenger RNA (mRNA), to stimulate an immune response against the virus. RNA vaccines offer certain advantages, including a faster production process compared to traditional methods.</p>
<p>Most influenza vaccines are typically produced using eggs. The process involves growing the virus in eggs and then inactivating it to create the vaccine. However, this method has limitations, and newer technologies, such as cell-based and recombinant vaccine production, are being developed to overcome these challenges.</p>
<p>Reverse genetics has emerged as a valuable tool in the manufacture of influenza virus vaccines. This technique allows scientists to manipulate the genetic material of the virus, facilitating the development of vaccines with desired characteristics. Reverse genetics has contributed to advancements in vaccine production, including the creation of candidate vaccines that can be tailored to match circulating strains more precisely.</p>
<section id="during-the-2009-h1n1-outbreak" class="level5" data-number="7.3.1.2.1">
<h5 data-number="7.3.1.2.1" class="anchored" data-anchor-id="during-the-2009-h1n1-outbreak"><span class="header-section-number">7.3.1.2.1</span> During the 2009 H1N1 Outbreak</h5>
<p>Vaccine production during the emergence of a new influenza virus, especially during a pandemic, poses challenges due to the limited time available to prepare sufficient vaccine stocks. A notable example is the pandemic of 2009, triggered by the emergence of a new influenza virus strain known as pH1N1/2009. The rapid emergence and global dissemination of this virus left little time to produce the necessary vaccine stocks to effectively combat the spread of the virus.</p>
<p>In response to the challenges posed by the pH1N1/2009 virus, alternative measures were employed to control its transmission. Antiviral drugs, such as Tamiflu, played a key role in managing and mitigating the impact of the pandemic. These drugs were used to treat infected individuals and help prevent further spread. Additionally, strategies such as virus detection and the quarantine of infected individuals were implemented to curb the transmission of the virus within communities.</p>
<p>The experience with the pH1N1/2009 pandemic underscored the importance of having multiple tools and strategies in place for the control of influenza viruses, especially when vaccine production may be constrained by time limitations during the early stages of a pandemic. Antiviral drugs, rapid detection methods, and effective public health measures remain crucial components of the response to emerging influenza viruses when timely vaccine production may not be feasible.</p>
</section>
</section>
</section>
<section id="bird-transmissions" class="level3" data-number="7.3.2">
<h3 data-number="7.3.2" class="anchored" data-anchor-id="bird-transmissions"><span class="header-section-number">7.3.2</span> Bird Transmissions</h3>
<p>Transmission by birds, especially aquatic birds, plays a significant role in the dynamics of influenza A viruses. All known subtypes of influenza A viruses have been identified in both domestic and wild aquatic birds. Among these birds, feral water birds, such as ducks and geese, are particularly noteworthy for hosting the largest number of influenza viruses.</p>
<p>In ducks, a substantial proportion of avian influenza strains replicate in the lungs and gastrointestinal tract. The virus is shed in nasal secretions and feces, contributing to its transmission. This shedding of the virus by infected birds becomes a key factor in the circulation of influenza A viruses. Ducks, being migratory waterfowl, play a crucial role in the continual circulation of influenza A viruses.</p>
<p>The natural reservoir of influenza A viruses in aquatic birds, combined with their migratory behaviors, creates an environment where these viruses can persist and spread. The ability of influenza A viruses to infect and be maintained in bird populations poses challenges for the prevention and control of avian influenza, especially considering the potential for the viruses to spill over into other species, including humans.</p>
<section id="in-birds" class="level4" data-number="7.3.2.1">
<h4 data-number="7.3.2.1" class="anchored" data-anchor-id="in-birds"><span class="header-section-number">7.3.2.1</span> In Birds?</h4>
<p>In ducks, influenza A virus infection typically does not cause severe illness, rendering the infection avirulent in these birds. This unique characteristic allows ducks to serve as a natural reservoir for influenza A viruses. Despite being carriers of the virus, ducks often do not show significant signs of illness, enabling the virus to persist within their populations.</p>
<p>However, the story changes when it comes to certain subtypes of the virus, specifically H5 and H7. In domestic poultry, these subtypes have the potential to produce systemic infections that can be far more severe. Systemic infection refers to the involvement of multiple organs and tissues throughout the body. In cases where domestic poultry, such as chickens or turkeys, are infected with H5 or H7 subtypes, high mortality rates can be associated with the infection.</p>
<p>The increased severity observed in domestic poultry contrasts with the avirulent nature of influenza A virus infection in ducks. This variation in the impact of different subtypes highlights the complexity of influenza viruses and the importance of understanding their behavior in different host species. It also underscores the potential risks associated with certain subtypes, especially those with the ability to cause systemic infection and higher mortality rates in domestic poultry.</p>
</section>
</section>
<section id="swine-transmission" class="level3" data-number="7.3.3">
<h3 data-number="7.3.3" class="anchored" data-anchor-id="swine-transmission"><span class="header-section-number">7.3.3</span> Swine Transmission</h3>
<p>Swine can play a role in the transmission of influenza viruses, being susceptible to infection with certain subtypes, notably H1N1 and H3N2. In swine populations, there are instances of classic swine influenza as well as viruses that are antigenically similar to those found in avian and human hosts.</p>
<p>The transmission of influenza from swine to humans has been demonstrated using serology techniques. This indicates that swine can serve as a source of influenza viruses that have the potential to infect humans. It’s important to note that H3N2 variants have been detected in pigs, and while these infections in pigs typically result in mild disease symptoms, the concern arises from the possibility of these viruses undergoing further changes and potentially causing infections in humans.</p>
<p>The ability of swine to be infected with influenza viruses from multiple sources, including avian and human viruses, adds to the complexity of influenza dynamics. This inter-species transmission can contribute to the emergence of novel influenza strains with the potential to infect humans. Monitoring and understanding influenza transmission in swine populations is crucial for assessing and mitigating the risks associated with the cross-species spread of influenza viruses.</p>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/week11/swine.png" class="img-fluid figure-img" width="447"></p>
<p></p><figcaption class="figure-caption">Swine Influenza Transmission</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p>Indeed, influenza variants with the potential to infect humans can emerge through reassortment events involving swine, avian, and human viruses. Swine are thought to act as a “mixing bowl” where different influenza viruses from various sources can undergo genetic reassortment.</p>
<p>In the close proximity of swine, which can be infected by both avian and human influenza viruses, the co-infection of these different strains provides an opportunity for their genetic material to mix and create novel virus variants. This reassortment can result in the emergence of influenza viruses with a combination of genetic elements from swine, avian, and human viruses.</p>
<p>The swine population’s ability to act as an intermediary host for influenza viruses from different sources underscores the importance of monitoring and understanding influenza dynamics in these animals. It also highlights the need for surveillance and control measures to prevent the emergence of novel influenza variants with the potential to cause infections in humans. The role of swine as a “mixing bowl” emphasizes the complex interactions between different influenza virus strains and the importance of addressing the risks associated with cross-species transmission.</p>
</section>
<section id="first-direct-transmission" class="level3" data-number="7.3.4">
<h3 data-number="7.3.4" class="anchored" data-anchor-id="first-direct-transmission"><span class="header-section-number">7.3.4</span> First Direct Transmission</h3>
<p>Direct transmission of avian influenza viruses to humans, without the involvement of an intermediate species like swine, was first reported in 1997. This transmission event involved the H5N1 subtype and occurred from infected poultry to humans. The H5N1 virus caused high mortality rates in both chickens (ranging from 70% to 100%) and humans (approximately 30%).</p>
<p>Key features of the H5N1 virus in this instance included a polybasic sequence in the hemagglutinin (HA) protein and a shortened stalk region in the neuraminidase (NA) protein. Despite causing severe disease in humans, there was no evidence of human-to-human transmission on this occasion.</p>
<p>This occurrence highlighted that avian influenza viruses have the potential to be directly transmitted to humans, resulting in illness. However, for efficient human-to-human spread, additional changes in the virus, such as mutations or reassortment events, are typically required. The ability of avian influenza viruses to directly infect humans underscores the importance of vigilant surveillance and control measures to monitor and mitigate the risks associated with these viruses. Additionally, understanding the factors that contribute to the adaptation and transmission of avian influenza viruses in humans is crucial for public health preparedness.</p>
</section>
</section>
<section id="host-adaptations" class="level2" data-number="7.4">
<h2 data-number="7.4" class="anchored" data-anchor-id="host-adaptations"><span class="header-section-number">7.4</span> Host Adaptations</h2>
<p>Host adaptation in influenza viruses involves various aspects of the virus particle adapting to replicate effectively in a specific host. This process implies that interactions between the virus and the host occur in a species-specific manner, meaning the virus is tailored to exploit the cellular environment of a particular host species.</p>
<p>The adaptation process includes interactions between virus proteins and host cell proteins, and these interactions are often mediated by specific amino acid sequences. The virus must evolve to recognize and interact with host cell factors that facilitate its entry, replication, and exit from the host cell. For example, the virus might need to recognize and bind to specific cell surface receptors on the host cell to initiate the infection process.</p>
<p>Key interactions occur between viral proteins, such as hemagglutinin (HA) and neuraminidase (NA), and host cell proteins. These interactions enable the virus to enter the host cell, replicate its genetic material, assemble new virus particles, and ultimately spread to infect other cells. The adaptation process often involves changes in these viral proteins that allow the virus to efficiently utilize the cellular machinery of the specific host species.</p>
<section id="influenza-polymerase-complex" class="level3" data-number="7.4.1">
<h3 data-number="7.4.1" class="anchored" data-anchor-id="influenza-polymerase-complex"><span class="header-section-number">7.4.1</span> Influenza Polymerase Complex</h3>
<p>The influenza virus polymerase complex is a fundamental component responsible for orchestrating the replication and transcription of the viral genome. Comprising three essential proteins—PB1, PB2, and PA—each protein within this complex has distinct functions vital for the virus’s lifecycle. The PA protein contributes endonuclease activity, a critical function involved in cleaving host cell mRNA to initiate viral mRNA synthesis. Additionally, the C-terminal domain of PA serves as an interaction site with the PB1 protein, fostering cohesion within the polymerase complex.</p>
<p>PB1, another key player, boasts polymerase activity essential for the synthesis of viral RNA. Its N-terminal region engages in a pivotal interaction with the PA protein, while the C-terminal region binds to the PB2 protein, fortifying the structural integrity of the complex. Moreover, PB1 directly binds to viral RNA, playing a crucial role in the initiation of viral RNA synthesis, a foundational step in the virus’s replication process.</p>
<p>In the intricate dance of the polymerase complex, the PB2 protein plays its part with a cap binding domain crucial for cap-snatching—a mechanism where the virus pilfers short fragments of capped RNA from host cell mRNA to prime viral mRNA synthesis. Additionally, PB2’s possession of a nuclear localization signal (NLS) guides the polymerase complex into the host cell nucleus, where the intricacies of viral replication and transcription unfold.</p>
<section id="mrna-synthesis" class="level4" data-number="7.4.1.1">
<h4 data-number="7.4.1.1" class="anchored" data-anchor-id="mrna-synthesis"><span class="header-section-number">7.4.1.1</span> mRNA Synthesis</h4>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/week11/virus_com.png" class="img-fluid figure-img" width="395"></p>
<p></p><figcaption class="figure-caption">Virus Complex</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p>In the context of mRNA synthesis during influenza virus replication, the polymerase complex plays a pivotal role and relies on the presence of viral RNA (vRNA) for activation. Within this complex, the PB1 subunit holds a critical function, housing the binding site for the 5’-terminal sequence of vRNA. The binding of this 5’-terminal sequence serves as a key activator, triggering the cap binding activity of the PB2 protein and initiating the PB1 binding activity at the 3’-terminal sequence of vRNA.</p>
<p>This sequential activation has a cascading effect on the polymerase complex’s functionality. The binding of the 3’-end of vRNA not only activates the cap binding activity of PB2 but also triggers an endonuclease activity in the PA protein. This endonuclease activity is essential for cleaving host cell mRNA, a crucial step in the cap-snatching process that provides primers for viral mRNA synthesis.</p>
<p>Ultimately, the nucleotide addition, a fundamental step in mRNA synthesis, is catalyzed by the PB1 protein. This orchestrated series of events highlights the intricate coordination within the influenza virus polymerase complex during mRNA synthesis. The interplay of these subunits and their activities showcases the adaptability of the virus in utilizing host cell machinery for its own replication, offering insights into potential targets for antiviral interventions.</p>
</section>
</section>
<section id="cap-snatching" class="level3" data-number="7.4.2">
<h3 data-number="7.4.2" class="anchored" data-anchor-id="cap-snatching"><span class="header-section-number">7.4.2</span> Cap Snatching</h3>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/week11/cap_snatching.png" class="img-fluid figure-img" width="226"></p>
<p></p><figcaption class="figure-caption">Cap Snatching</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p>Cap snatching is a distinctive mechanism employed by influenza viruses during their replication cycle. Early studies demonstrated that the synthesis of viral mRNA is hindered by α-amanitin, an inhibitor targeting RNA polymerase II (RNA pol II), a key enzyme involved in transcribing cellular genes. This inhibition underscores the dependence of influenza virus replication on the host-cell nuclear machinery.</p>
<p>In the cap-snatching process, the virus hijacks host cell transcripts to acquire primers for the synthesis of its mRNA. Specifically, the primers are short capped RNA fragments denoted as m7 GpppXm, where X represents various nucleotides. These fragments are originally generated by the activity of RNA polymerase II in the host cell. The term “cap snatching” aptly describes this unique viral strategy, where the influenza virus seizes these capped fragments from host transcripts to kickstart its own mRNA synthesis.</p>
<p>This distinctive feature sets influenza apart from other viruses and showcases the virus’s ability to co-opt host cell components for its own replication. Understanding cap snatching provides insights into potential vulnerabilities in the viral life cycle and offers avenues for the development of antiviral strategies aimed at disrupting this specific mechanism.</p>
</section>
<section id="host-protein-mutations" class="level3" data-number="7.4.3">
<h3 data-number="7.4.3" class="anchored" data-anchor-id="host-protein-mutations"><span class="header-section-number">7.4.3</span> Host Protein Mutations</h3>
<p>Host adaptation of influenza virus P proteins, particularly mutations associated with avian-to-human adaptation, has been a subject of interest in influenza research. Notable mutations like E627K, T271A, E158G, and D701M have been identified in these proteins. While the molecular basis of these mutations in terms of species adaptation remains uncertain in general, some insights have been gleaned from specific cases.</p>
<p>For instance, the E627K mutation in the PB2 protein has been implicated in host adaptation. This mutation is thought to enable high replication rates at the lower temperatures typically found in human respiratory tracts compared to avian hosts. Additionally, changes in the sequence may influence interactions between the PB2 protein and the NP (nucleoprotein) protein, suggesting a potential mechanism for adaptation.</p>
<p>Furthermore, the involvement of ANP32A, a host cellular protein, has been considered in the context of host adaptation. Sequence changes, such as those observed in the identified mutations, may confer differences in the interaction of influenza P proteins with cellular proteins, potentially influencing the virus’s ability to replicate and adapt to new host species.</p>
<p>While the precise molecular mechanisms underlying these mutations and their roles in host adaptation are not fully elucidated, ongoing research aims to unravel the intricate interplay between viral proteins and host factors. Understanding these interactions is crucial for predicting and mitigating the risks associated with the emergence of influenza strains with increased potential for human transmission.</p>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/week11/pb2.png" class="img-fluid figure-img" width="170"></p>
<p></p><figcaption class="figure-caption">PB2 Protein</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p>Indeed, other sequence changes in the PB2 protein of influenza virus have been implicated in interactions with host cellular components, particularly importins. One notable example is the D701M mutation in the PB2 protein. This mutation has been found to enhance the binding of the PB2 protein to mammalian importins.</p>
<p><strong>Importins</strong> are cellular proteins that play a crucial role in the transport of molecules, including viral proteins, into the nucleus of host cells. The D701M mutation appears to influence the efficiency and specificity of these interactions, potentially facilitating the transport of viral components into the nucleus of mammalian cells more effectively.</p>
</section>
<section id="receptor-usage" class="level3" data-number="7.4.4">
<h3 data-number="7.4.4" class="anchored" data-anchor-id="receptor-usage"><span class="header-section-number">7.4.4</span> Receptor Usage</h3>
<p>The receptor usage of influenza viruses is a crucial factor in determining their ability to infect different species. Human influenza viruses, particularly H3 strains, predominantly use sialyoliogosaccharides terminated by α2,6 galactose for binding, whereas avian influenza viruses favor binding to α2,3 galactose.</p>
<p>This receptor specificity is dictated by the structure of the hemagglutinin (HA) binding pocket, a viral protein that plays a key role in the initial stages of viral entry into host cells. Avian intestines primarily contain α2,3 linkages, pig trachea features both α2,3 and α2,6 linkages, and human trachea predominantly has α2,6 linkages. This structural variation in sialic acid linkages acts as a determinant for the specificity of the virus, influencing its ability to transfer between different species.</p>
<p>For example, the preference for α2,3 linkages in avian viruses serves as a restriction for direct transmission to humans, as human respiratory tissues predominantly feature α2,6 linkages. Pigs, however, play a unique role as potential intermediate hosts, as they possess both human (α2,6 galactose) and avian (α2,3 galactose) receptors. This characteristic makes swine a significant host in the interspecies transmission of influenza viruses, potentially facilitating the adaptation of avian viruses to human hosts. Understanding these receptor-specific interactions is crucial for predicting and monitoring the emergence of influenza strains with pandemic potential.</p>
</section>
<section id="activating-ha-protein" class="level3" data-number="7.4.5">
<h3 data-number="7.4.5" class="anchored" data-anchor-id="activating-ha-protein"><span class="header-section-number">7.4.5</span> Activating HA Protein</h3>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../imgs/week11/cleavage_sites.png" class="img-fluid figure-img" width="334"></p>
<p></p><figcaption class="figure-caption">Cleavage Sites of the HA Protein</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p>The hemagglutinin (HA) protein is a central player in the pathogenicity of influenza viruses. Its activation is a crucial step, involving protease cleavage that transforms the precursor form (HA0) into the mature protein consisting of HA1 and HA2 subunits.</p>
<p>The cleavage site of HA is notable for its basic residues, and this feature plays a significant role in determining the activation process. There are two main types of cleavage sites based on the number of basic residues present:</p>
<ol type="1">
<li><p><strong>Polybasic Sequence (4-6 basic residues):</strong> This type of cleavage site is cleaved by enzymes such as furin and furin-like enzymes, which are expressed in various cell types. The presence of these enzymes in a wide range of cells contributes to the broad tissue tropism of influenza viruses. The cleavage by furin-like enzymes is associated with a higher pathogenic potential.</p></li>
<li><p><strong>Single/Few Basic Residue:</strong> In this case, cleavage occurs through specific proteases found in particular cell types. For example, the Clara protease is a specific protease found in specialized respiratory cells. The specificity of cleavage in certain cell types can influence the tissue tropism and pathogenicity of the virus.</p></li>
</ol>
<p>The nature of the cleavage site is a key determinant in the classification of influenza viruses as avirulent or virulent. Viruses with a polybasic cleavage site are often associated with increased pathogenicity, as they can be cleaved by a broader range of host enzymes, allowing for more efficient viral replication and spread. Understanding the nuances of HA proteolytic activation provides insights into the factors influencing the pathogenic potential of influenza viruses.</p>


</section>
</section>

</main> <!-- /main -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const disableStylesheet = (stylesheets) => {
    for (let i=0; i < stylesheets.length; i++) {
      const stylesheet = stylesheets[i];
      stylesheet.rel = 'prefetch';
    }
  }
  const enableStylesheet = (stylesheets) => {
    for (let i=0; i < stylesheets.length; i++) {
      const stylesheet = stylesheets[i];
      stylesheet.rel = 'stylesheet';
    }
  }
  const manageTransitions = (selector, allowTransitions) => {
    const els = window.document.querySelectorAll(selector);
    for (let i=0; i < els.length; i++) {
      const el = els[i];
      if (allowTransitions) {
        el.classList.remove('notransition');
      } else {
        el.classList.add('notransition');
      }
    }
  }
  const toggleColorMode = (alternate) => {
    // Switch the stylesheets
    const alternateStylesheets = window.document.querySelectorAll('link.quarto-color-scheme.quarto-color-alternate');
    manageTransitions('#quarto-margin-sidebar .nav-link', false);
    if (alternate) {
      enableStylesheet(alternateStylesheets);
      for (const sheetNode of alternateStylesheets) {
        if (sheetNode.id === "quarto-bootstrap") {
          toggleBodyColorMode(sheetNode);
        }
      }
    } else {
      disableStylesheet(alternateStylesheets);
      toggleBodyColorPrimary();
    }
    manageTransitions('#quarto-margin-sidebar .nav-link', true);
    // Switch the toggles
    const toggles = window.document.querySelectorAll('.quarto-color-scheme-toggle');
    for (let i=0; i < toggles.length; i++) {
      const toggle = toggles[i];
      if (toggle) {
        if (alternate) {
          toggle.classList.add("alternate");     
        } else {
          toggle.classList.remove("alternate");
        }
      }
    }
    // Hack to workaround the fact that safari doesn't
    // properly recolor the scrollbar when toggling (#1455)
    if (navigator.userAgent.indexOf('Safari') > 0 && navigator.userAgent.indexOf('Chrome') == -1) {
      manageTransitions("body", false);
      window.scrollTo(0, 1);
      setTimeout(() => {
        window.scrollTo(0, 0);
        manageTransitions("body", true);
      }, 40);  
    }
  }
  const isFileUrl = () => { 
    return window.location.protocol === 'file:';
  }
  const hasAlternateSentinel = () => {  
    let styleSentinel = getColorSchemeSentinel();
    if (styleSentinel !== null) {
      return styleSentinel === "alternate";
    } else {
      return false;
    }
  }
  const setStyleSentinel = (alternate) => {
    const value = alternate ? "alternate" : "default";
    if (!isFileUrl()) {
      window.localStorage.setItem("quarto-color-scheme", value);
    } else {
      localAlternateSentinel = value;
    }
  }
  const getColorSchemeSentinel = () => {
    if (!isFileUrl()) {
      const storageValue = window.localStorage.getItem("quarto-color-scheme");
      return storageValue != null ? storageValue : localAlternateSentinel;
    } else {
      return localAlternateSentinel;
    }
  }
  let localAlternateSentinel = 'default';
  // Dark / light mode switch
  window.quartoToggleColorScheme = () => {
    // Read the current dark / light value 
    let toAlternate = !hasAlternateSentinel();
    toggleColorMode(toAlternate);
    setStyleSentinel(toAlternate);
  };
  // Ensure there is a toggle, if there isn't float one in the top right
  if (window.document.querySelector('.quarto-color-scheme-toggle') === null) {
    const a = window.document.createElement('a');
    a.classList.add('top-right');
    a.classList.add('quarto-color-scheme-toggle');
    a.href = "";
    a.onclick = function() { try { window.quartoToggleColorScheme(); } catch {} return false; };
    const i = window.document.createElement("i");
    i.classList.add('bi');
    a.appendChild(i);
    window.document.body.appendChild(a);
  }
  // Switch to dark mode if need be
  if (hasAlternateSentinel()) {
    toggleColorMode(true);
  } 
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const clipboard = new window.ClipboardJS('.code-copy-button', {
    target: function(trigger) {
      return trigger.previousElementSibling;
    }
  });
  clipboard.on('success', function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    setTimeout(function() {
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  });
  function tippyHover(el, contentFn) {
    const config = {
      allowHTML: true,
      content: contentFn,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start'
    };
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      return note.innerHTML;
    });
  }
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const cites = ref.parentNode.getAttribute('data-cites').split(' ');
    tippyHover(ref, function() {
      var popup = window.document.createElement('div');
      cites.forEach(function(cite) {
        var citeDiv = window.document.createElement('div');
        citeDiv.classList.add('hanging-indent');
        citeDiv.classList.add('csl-entry');
        var biblioDiv = window.document.getElementById('ref-' + cite);
        if (biblioDiv) {
          citeDiv.innerHTML = biblioDiv.innerHTML;
        }
        popup.appendChild(citeDiv);
      });
      return popup.innerHTML;
    });
  }
});
</script>
<nav class="page-navigation">
  <div class="nav-page nav-page-previous">
      <a href="../chapters/week6.html" class="pagination-link">
        <i class="bi bi-arrow-left-short"></i> <span class="nav-page-text"><span class="chapter-number">6</span>&nbsp; <span class="chapter-title">Antiviral Strategies and Virus Vaccines</span></span>
      </a>          
  </div>
  <div class="nav-page nav-page-next">
      <a href="../chapters/week2.html" class="pagination-link">
        <span class="nav-page-text"><span class="chapter-number">8</span>&nbsp; <span class="chapter-title">Seeing Viruses</span></span> <i class="bi bi-arrow-right-short"></i>
      </a>
  </div>
</nav>
</div> <!-- /content -->



</body></html>